hle-survey/transformation/src/test/scala/org/broadinstitute/monster/dap/DogTransformationsSpec.scala

```diff
@@ -5,28 +5,601 @@ import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 import java.time.LocalDate
 
-class DogTransformationsSpec extends AnyFlatSpec with Matchers {
+import org.scalatest.OptionValues
+
+class DogTransformationsSpec extends AnyFlatSpec with Matchers with OptionValues {
   behavior of "DogTransformations"
 
-  // TODO add tests for dog demographics transformation
+  it should "map required fields" in {
+    val mapped = DogTransformations.mapDog(RawRecord(1, Map("st_owner_id" -> Array("2"))))
+    mapped shouldBe HlesDog.init(1L, 2L)
+  }
 
-  it should "correctly map study status fields" in {
+  it should "map study status fields" in {
     val exampleDogFields = Map[String, Array[String]](
       "st_vip_or_staff" -> Array("2"),
       "st_batch_label" -> Array("this is my label"),
-      "st_invite_to_portal" -> Array("05-22-2020"),
-      "st_portal_account_date" -> Array("01-01-2000"),
-      "st_dap_pack_date" -> Array("12-31-1999")
+      "st_invite_to_portal" -> Array("2020-05-22"),
+      "st_portal_account_date" -> Array("2000-01-01"),
+      "st_dap_pack_date" -> Array("2020-01-15 10:21")
```

Comment:
Oh, so they are the standard format! I take it you were able to find an example of one of the dates then?

Reply:
Yes, there was a bug on the RedCap side which flagged all dates as PII. They fixed it after our last call with them.